src/RaceChrono.cpp

```diff
@@ -39,18 +44,71 @@ void handle_racechrono_filter_request(
 
   // TODO: figure out how to report errors.
 }
+#elif defined(ARDUINO_ARCH_ESP32)
+class handle_racechrono_filter_request: public NimBLECharacteristicCallbacks {
```

Comment:
Please use CamelCase for class names to be consistent with the rest of the code.

I try to follow https://google.github.io/styleguide/cppguide.html whenever it doesn't contradict Arduino conventions.

Reply:
That's what I get when I "borrow" code willy-nilly :)